include/avif/avif.h

```diff
@@ -1312,16 +1312,21 @@ typedef struct avifDecoder
     // Version 1.1.0 ends here. Add any new members after this line.
 
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
-    // This is true when avifDecoderParse() detects a gain map.
+    // If 'enableParsingGainMapMetadata' is false: this is true when avifDecoderParse()
+    // detects a gain map.
+    // If 'enableParsingGainMapMetadata' is true: this is true when avifDecoderParse()
+    // detects a gain map whose metadata is a supported version.
```

Comment:
Nit/optional: It seems clearer to move the `gainMapPresent` field after the `enableParsingGainMapMetadata` field.

We can move `gainMapPresent` after `ignoreColorAndAlpha`, so that we first declare the input options, and then declare the output options.